tests/tests_integration_pytest.yml

```diff
@@ -0,0 +1,66 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- name: Empty play to gather facts
+  hosts: all
+
+- import_playbook: playbooks/centos7_deps.yml
+  when:
+    - ansible_distribution == "CentOS"
+    - ansible_distribution_major_version >= '7'
+
+- import_playbook: playbooks/fedora32_deps.yml
+  when:
+    - ansible_distribution == "Fedora"
+    - ansible_distribution_major_version == '32'
+
+- name: Run Pytest tests
```

Comment:
It might be good to move this into its own playbook and import it only for the already supported distributions, CentOS 8 and Fedora 32 to avoid the failures on the other distributions.

Reply:
Done, thanks!